### ISSUE TITLE
Brightness shows as 39% instead of 100%

### ISSUE THREAD

**Issue by BlueFireTV** [bwp91/homebridge-govee#466]
### What issue do you have? Please be as thorough and explicit as possible.
When i set the brightness to 100% it is at 100% but it shows as 39% for a few seconds and dann it shows as 100%

### Details of your setup.
* Do you use Homebridge (with config-ui?) or HOOBS? 
homebridge-config-ui-x v4.50.2

* Which version of Homebridge/HOOBS do you have?
homebridge-config-ui-x v4.50.2

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?

homebridge-govee v8.0.0-beta.4 

* Which Govee devices do you have that are causing issues? Please include product models if applicable.

H619E

### Please paste any relevant logs below.

```
[3/11/2023, 10:12:20 PM] [Govee] [Licht] current brightness [100%].
[3/11/2023, 10:12:20 PM] [Govee] [Licht] current brightness [39%].
[3/11/2023, 10:12:33 PM] [Govee] [Licht] current brightness [100%].
```

**Comment by pgogins1** [comment]
Same issue I'm having.  Any updates on this?

**Comment by bwp91** [comment]
If when you choose 100% in the govee app and it shows as 39% in the home app then please do the following:

in the plugin settings screen in the light devices section, enter the device id and a label to show the rest of the options, then enable the aws brightness no scale option, then restart homebridge.

**Comment by pgogins1** [comment]
I wasn't able to reference the "aws brightness no scale option".  It's seems I do t have that option.

**Comment by pgogins1** [comment]
Any updates?

**Comment by bwp91** [comment]
![Screenshot 2023-04-10 at 14 13 36](https://user-images.githubusercontent.com/43026681/230907878-dbb5e70e-3144-4142-91fd-afdb604098ff.png)

The option is here

**Comment by pgogins1** [comment]
For whatever reason, I don't have that option.  

I'm running 8.1.1 
 
Node 16.9.1

Homebridge 1.6.0

Label and device ID are inserted.

**Comment by bwp91** [comment]
Can you show a screenshot of your settings screen?

**Comment by pgogins1** [comment]
Best Regards!On Apr 10, 2023, at 9:45 AM, Ben ***@***.***> wrote:﻿
Can you show a screenshot of your settings screen?

**Comment by bwp91** [comment]
Not sure if you attached something, but i don’t think it appears here if you attach something by email

**Comment by pgogins1** [comment]
![image](https://user-images.githubusercontent.com/129900582/230926172-415b20a4-421d-4298-ab7c-37342b2b50e1.jpeg)

![image](https://user-images.githubusercontent.com/129900582/230926211-0e92f7af-6875-4cfd-bb05-2053cfef5c5f.jpeg)

![image](https://user-images.githubusercontent.com/129900582/230926252-eb23ae83-6183-48d6-a367-69a161cfeee3.jpeg)

**Comment by pgogins1** [comment]
Okay, I sent it through GitHub.Best Regards!On Apr 10, 2023, at 9:49 AM, Ben ***@***.***> wrote:﻿
Not sure if you attached something, but i don’t think it appears here if you attach something by email

**Comment by bwp91** [comment]
Ah you are using HOOBS.
They save a copy of some plugin settings screens so even when I update the plugin with new options (and remove old options) these can not show up (or old ones still appear) in the HOOBS UI.
You will need to ask HOOBS to update the settings config for this plugin 🤨

**Comment by pgogins1** [comment]
🤦🏽‍♂️Okay, will do! Thanks.

**Comment by pgogins1** [comment]
![image](https://user-images.githubusercontent.com/129900582/230967407-ced6fe60-6739-47b6-bf94-d53920e357c7.png)

**Comment by pgogins1** [comment]
This is the feedback from Hoobs

![image](https://user-images.githubusercontent.com/129900582/231528292-88528863-5967-4f65-b4bd-e03037a42593.jpeg)

**Comment by pgogins1** [comment]
![image](https://user-images.githubusercontent.com/129900582/231532836-ce1c1442-6bcd-4dbc-81ac-4f354ca3be65.png)

**Comment by bwp91** [comment]
next thing to try:

> you have to clear your hub cache from hub settings to see it and you might need to restart the HOOBS app

**Comment by bwp91** [comment]
any luck?

**Comment by pgogins1** [comment]
Cache was cleared. No luck

**Comment by bwp91** [comment]
Have contacted hoobs myself, will let you know 👍🏻

**Comment by pgogins1** [comment]
Nice work!  Configuration settings are now available.

**Comment by bwp91** [comment]
have you managed to get this fixed now?

**Comment by pgogins1** [comment]
Yes I did! Works perfectly now.Best Regards!On Apr 14, 2023, at 12:38 PM, Ben ***@***.***> wrote:﻿
have you managed to get this fixed now?

**Comment by bwp91** [comment]
Awesome.
Gonna close this issue as although OP has not made any further comments, the solution is on the wiki page 👍🏻 any further issues please do create a new issue

**Comment by pgogins1** [comment]
Absolutely!Best Regards!On Apr 14, 2023, at 3:01 PM, Ben ***@***.***> wrote:﻿
Closed #466 as completed.
